src/redux/Categories/Categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] It would be better if you write your action type following the [ducks pattern](https://github.com/erikras/ducks-modular-redux). 

```suggestion
const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
```